src/Components/LoginApp.css

```diff
@@ -3,7 +3,7 @@
 }
 
 .background_login {
-  background-image: url(/Recess_Bg.jpg);
+  background-image: url(/Recess_Bg_2.jpg);
```

Comment:
This jpg does not seem to exist in this pull request.  Can you add it in?

Reply:
added